apps/ui/src/composables/useTownhall.ts

```diff
@@ -259,6 +259,38 @@ export function useTownhall() {
     );
   }
 
+  async function sendClaimRole(space: string, id: string) {
+    if (!auth.value) {
+      throw new Error('Not authenticated');
+    }
+
+    const signer = await getAliasSigner(auth.value.provider);
+
+    return wrapPromise(
+      highlightClient.claimRole({
+        signer,
+        data: { space, id },
+        salt: getSalt()
+      })
+    );
+  }
+
+  async function sendRevokeRole(space: string, id: string) {
+    if (!auth.value) {
+      throw new Error('Not authenticated');
```

Comment:
Would be great to open login modal instead (similar than with others SX / S actions)